collection-scripts/gather_dr_resources

```diff
@@ -53,3 +53,13 @@ for NAMESPACE in "${NAMESPACES[@]}"; do
     # shellcheck disable=SC2086
     { oc describe configmap -n ${NAMESPACE}; } >>"${COMMAND_OUTPUT_FILE}"
 done
+
+# Collect logs for all pods named "volsync" across all namespaces
+dbglog "Starting volsync pod logs collection..."
+
+oc get pods --all-namespaces | grep volsync | while read -r ns pod _; do
```

Comment:
please check the code above, I would suggest to use same for loop format to maintain the consistency in the code.